sw/host/opentitantool/src/backend/mod.rs

```diff
@@ -38,12 +43,27 @@ pub enum Error {
 }
 
 /// Creates the requested backend interface according to [`BackendOpts`].
-pub fn create(args: &BackendOpts) -> Result<Box<dyn Transport>> {
-    match args.interface.as_str() {
-        "" => Ok(Box::new(EmptyTransport)),
+pub fn create(args: &BackendOpts) -> Result<TransportWrapper> {
+    let mut env = TransportWrapper::new(match args.interface.as_str() {
+        "" => Ok(Box::new(EmptyTransport) as Box<dyn Transport>),
```

Comment:
nit: maybe we should add a `new` or `create` method to `EmptyTransport` just to make the match arms all look regular.